signals/main.py

```diff
@@ -24,9 +25,19 @@ def run_main(schema, generator_name, data_models, core_data, project_name, api_u
     except SignalsError as e:
         print(str(e))
     else:
+        clear_generated_code_files()
```

Comment:
Also, we may want to do this under `finally` instead of `else`. If Signals gets an error in the `except` clause then clearing the generated files won't get ran and we may still leave the folder there. I believe `finally` gets run whether or not it errored.